File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsGoodsTypeMapper.java
package top.charjin.shoppingserver.mapper;

import top.charjin.shoppingserver.entity.OsGoodsType;

public interface OsGoodsTypeMapper {
    int deleteByPrimaryKey(Integer id);

    int insert(OsGoodsType record);

    int insertSelective(OsGoodsType record);

    OsGoodsType selectByPrimaryKey(Integer id);

    int updateByPrimaryKeySelective(OsGoodsType record);

    int updateByPrimaryKey(OsGoodsType record);
}

File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsGoodsTypeMapper.kt
package top.charjin.shoppingserver.mapper

import top.charjin.shoppingserver.entity.OsGoodsType

interface OsGoodsTypeMapper {
    fun deleteByPrimaryKey(id: Int?): Int

    fun insert(record: OsGoodsType): Int

    fun insertSelective(record: OsGoodsType): Int

    fun selectByPrimaryKey(id: Int?): OsGoodsType

    fun updateByPrimaryKeySelective(record: OsGoodsType): Int

    fun updateByPrimaryKey(record: OsGoodsType): Int
}